src/components/UI/Card/Card.js

```diff
@@ -52,6 +61,29 @@ export default function Card({ title, date, description, image, id, remove }) {
       dispatch(remove(id))
    }
 
+   const [editModal, setEditModal] = React.useState(false)
+
+   const [editModalData, setEditModalData] = React.useState(initModalState)
+
+   const editCardName = (e) =>
```

Comment:
You can use one handler for a lot of inputs 
[here](https://www.pluralsight.com/guides/handling-multiple-inputs-with-single-onchange-handler-react)

Reply:
thanks!